app/spec/controllers/cbv/generic_links_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe Cbv::GenericLinksController do
+  describe '#show' do
+    context 'when the hostname matches a client agency domain' do
+      it do
```

Comment:
Let's name our specs as long as they are multi-line.

```suggestion
      it "starts a CBV flow" do
```